src/main/java/net/atos/zac/app/zaken/model/RESTZaakAanmaakGegevens.java

```diff
@@ -7,10 +7,14 @@
 
 import java.util.List;
 
+import jakarta.validation.Valid;
+
 import net.atos.zac.app.bag.model.RESTBAGObject;
 import net.atos.zac.app.productaanvragen.model.RESTInboxProductaanvraag;
 
 public class RESTZaakAanmaakGegevens {
+
+    @Valid
```

Comment:
This `@Valid` is really required here? Seems a bit cumbersome..

Reply:
Ok. Let's leave it for now. Maybe look at this later. I hope we can make it a bit simpler somehow. :-)